docs/usability/ipfs-webui-useabilty-test.md

```diff
@@ -0,0 +1,82 @@
+- Find the total size of your IPFS repo _missing_
+
+More holistically, it should give a feeling of being connected to a network of peers.
+It must not give a sense of "uploading to the cloud", as that is a significant cause
+of confusion about how IPFS works, and it's dangerously misleading.
+
+The app ensure that the following actions are **possible** to complete
+
+- Determine the peer ID of the node
+- Determine how it is connected to IPFS
+- Find which network addresses it's bound to
+
+## Goal of the usability test
+
+- Find any problems prevent any of the participants from completing the tasks
+- Gather structured feedback on the v2 redesign
+
+## Methods
+
+- 45 min interview with PL folk, some developers, some not.
+- Notes, no recording
+- Remote or in-person
+- Include a retrospective design crit
+
+## Test Script
+
+- Visit webui.ipfs.io and find the peer ID of your node
+  _Do they have a daemon running already?_
+  _Do they follow the prompts to start their daemon?_
+  _Do they have to go and download go-ipfs first?_
+
+- Add a picture you'd like to share to your IPFS node
+- Copy the share link and send it to me via (whatever works best for both)
+  - _Do they figure out where the add files feature is?_
+  - _Do they manage to upload the file?_
+  - _Do they find and copy the share link?_
+  - _Do they try drag-and-drop? or do they use the add button or..._
+  - _Are you able to get the file from ipfs webui at your end_
+
+- I'll send you a share link. Download the file to your desktop
+  - _Do they find the download link?_
+  - _Do they get the file?_
+
+- Add the file and and explore it the IPLD data behind it. What node type is it?
+
+- Explore the "Project Apollo Archives" IPLD dataset, follow the "albums" link, and find picture to view on the IPFS gateway
+  - _Do they click through the album link?_
+  - _Do they find a jpg?_
+  _ _Do they find the View on IPFS Gateway link?_
+
+- Pick a dataset from https://archives.ipfs.io/ and explore it the IPLD structure behind it.
+  - _Do they? ...this requires copy and pasting the cid from the website into the explore form_
+
+- How many peers is your node connected to?
+  _Do they find out? How do they feel about that?_
+
+
+## Questions / Musings
+
+- How do we test for discoverability? (https://github.com/ipfs-shipyard/ipfs-webui/issues/815)
+  - how does a new user even find the webui? The daemon doesn't log out an address for it on start up.
+  - If we test for that with new users, I think we'd find most wouldn't know where to look.
+  - this is probably just another UX issue, and we need to fix it. I think I'm loathed to test it right now as the UX is so bad it's not worth testing yet.
+
+- How do we check for mis-conceptions the app might dispell or reinforce?
+  - IPFS is not a Backup / upload to the cloud service. It's still up to you to ensure your files are on more than one machine.
```

Comment:
It would be interesting to ask people (1) to describe how long data uploaded via webui or share.ipfs.io will be available [on public gateway], and (2) why they think what they think.

Reply:
These are really good areas of understanding that we need to dive into! Yays.

I really like your suggested question (1) @lidel ! We should find a different way of getting at (2) tho -- humans aren't generally very good at accurately describing why they do something or know something because they'll naturally try to give you an answer that makes them look smart, or gives you the answer they think *you* want, because they want to seem likable. Human nature n such. So we shall have to be sneaky.

In any case tho....hmmmmmm........as a first step, I like the idea of adding a version of @lidel 's question to the test script. After they send the link to the interviewer, could ask something like:
- How long do you think that picture will still be available/at that address/at that link?
- _If they answer so that it seems like they understand about the close-the-computer-its-unavailable thing, could ask_ How would you make sure that picture is always available?
- Can anyone find that photo, or just me?

You can likely improve on those questions, but that's a start -- what do you think?

For bigger questions of understanding in general, it might be good to decide what the Mission of the App is (you might know this already, and I just don't know the latest!). Is it meant to be someone's first toe-dip into IPFS-land? Or...? Having this target in mind will help understand what sort of research we need to understand potential misconceptions, or even at the start, what the app should be helping folks understand. ie, I could see a different sort of module or illustration that we should develop, or metric, if one of the key communication points we want to convey is that IPFS is not a backup service (something around time, perhaps, and how that interacts with IPFS). And if we want to convey that 'uploads' are technically available to anyone on the network, I could see some different ways of illustrating that, too.

But maybe those shouldn't be main 'talking points' for the app in the first place. But! It could be really cool to make the app the place people go to understand how IPFS works, soooooo.....hmmmmm... 🤔